Description: Modify paths for components of brahms
 In order to be more Debian friendly.
Author: Seb James <seb.james@sheffield.ac.uk>
Forwarded: No need to forward
Last-Update: 2014-02-21
--- a/framework/engine/support/os.h
+++ b/framework/engine/support/os.h
@@ -71,10 +71,10 @@
 		const char ENV_SEPARATOR[] = ":";
 		const char COMPONENT_DLL_EXT[] = "so";
 		const char COMPONENT_MODULE_FORM_OLD[] = "libbrahms-((CLASS_US)).so.0";
-		const char M_BINDING_MODULE_FORM[] = "/BRAHMS/bindings/component/1258/libbrahms-1258.so.0";
-		const char PY_BINDING_MODULE_FORM[] = "/BRAHMS/bindings/component/1262/libbrahms-1262.so.0";
-		const char CHANNEL_MODULE_FORM[] = "/BRAHMS/bin/libbrahms-channel-((PROTOCOL)).so";
-		const char COMPRESS_MODULE_FORM[] = "/BRAHMS/bin/libbrahms-compress.so";
+		const char M_BINDING_MODULE_FORM[] = "/usr/lib/brahms/bindings/component/1258/libbrahms-1258.so.0";
+		const char PY_BINDING_MODULE_FORM[] = "/usr/lib/brahms/bindings/component/1262/libbrahms-1262.so.0";
+		const char CHANNEL_MODULE_FORM[] = "/usr/lib/brahms/libbrahms-channel-((PROTOCOL)).so";
+		const char COMPRESS_MODULE_FORM[] = "/usr/lib/brahms/libbrahms-compress.so";
 
 	#endif
 
--- a/framework/engine/main/engine.cpp
+++ b/framework/engine/main/engine.cpp
@@ -178,7 +178,7 @@
 
 				//	load install/machine/user-level environment
 				{ FOUT_SECTION("Install-Level")
-					engineData.environment.load(brahms::os::getenv("SYSTEMML_INSTALL_PATH") + "/BRAHMS/brahms.xml", true, fout);
+					engineData.environment.load(brahms::os::getenv("SYSTEMML_INSTALL_PATH") + "/usr/share/brahms/brahms.xml", true, fout);
 				}
 				{ FOUT_SECTION("Machine-Level")
 					engineData.environment.load(brahms::os::getspecialfolder("appdata.machine") + "/brahms.xml", false, fout);
